packages/angular/src/generators/stories/schema.d.ts

```diff
@@ -1,5 +1,6 @@
 export interface StoriesGeneratorOptions {
   name: string;
+  interactionTests?: boolean;
   cypressProject?: string;
   generateCypressSpecs?: boolean;
```

Comment:
Let's add a deprecation on the types so it appears for anyone calling this generator from their own generator.
```suggestion
  /**
   * @deprecated Use Storybook interaction tests instead. It will be removed in v18.
   */
  cypressProject?: string;
  /**
   * @deprecated Use Storybook interaction tests instead. It will be removed in v18.
   */
  generateCypressSpecs?: boolean;
```